### ISSUE TITLE
Publish the plugin: This is more than WIP

### ISSUE THREAD

**Issue by prateekmedia** [simrat39/flgtk#1]
I think this is ready to be published, also `system_theme` plugin needs this to fetch theme for linux, So it would be good if we publish this.

https://github.com/bdlukaa/system_theme/issues/2

**Comment by simrat39** [comment]
yep, just need some docs and comments and i'll do it by tomorrow

**Comment by prateekmedia** [comment]
@simrat39 Are you still working on this? You can tell me what needs to be done if you are busy.
